crates/metadata/lib.rs

```diff
@@ -277,6 +281,11 @@ impl Metadata {
         map.insert("DOCS_RS", "1".into());
         map
     }
+
+    /// Return the custom docker image, if provided.
+    pub fn docker_image(&self) -> Option<String> {
+        self.docker_image.clone()
```

Comment:
Let's return a borrow here, since it's not dynamic.
```suggestion
    pub fn docker_image(&self) -> Option<&str> {
        self.docker_image.as_ref()
```